contracts/hackatom/tests/integration.rs

```diff
@@ -249,7 +247,7 @@ fn execute_release_works() {
     );
     assert_eq!(
         execute_res.attributes,
-        vec![attr("action", "release"), attr("destination", "benefits")],
+        vec![("action", "release"), ("destination", "benefits")],
```

Comment:
Huh? I really need to look at the std changes. When does it get converted to String?

Reply:
It doesn't! I implemented equality tests between tuples of string-y types and Attributes. Just for testing convenience.

Reply:
I saw that later. Very awesome